charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.43.0
+version: 9.43.1
```

Comment:
I don't think updating docs via helm-docs would satisfy the need to cut a new patch version, so let's skip this?

Reply:
@jackfrancis, if we don't update the helm chart version then `Lint and Test Charts / Helm chart (pull_request)` will fail.

because we use automated tooling to build and publish the releases of the helm chart, here is the workflow link:
https://github.com/kubernetes/autoscaler/blob/master/.github/workflows/release.yaml#L4-L29

Reply:
That settles it then!